### PR TITLE
Added alternative mods suggestions

### PR DESCRIPTION
**Pull request by PoorPocketsMcNewHold**
### Added Kilul Animation packs & tf2 bot kicker gui as alternative suggestions.

- [TF2 Bot kicker gui](https://github.com/Bash-09/tf2-bot-kicker-gui) has been my go-to tool for cheater/bot kicking on Linux. Has the PazerOP tool [had issues being worked on porting it to Linux](https://github.com/PazerOP/tf2_bot_detector/issues/56), this is a great contender, and can even be preferred for Windows users too *(if they are a rust fanboy)*.
- [Kilul Animation packs](https://gamebanana.com/members/1655507) has been an animator which has begun to gain some success for their animations on gamebanana, especially since they prioritized on doing animation for classes which [has been reported has having outdated broken inspection animation by Paysus himself](https://steamcommunity.com/groups/PaysusSkins). This make for Kilul packs, are great option to use for those classes instead of waiting for an updated from Paysus and having to rely on broken animation mods. 

I do want to add also some TF2 vanilla HUD's such as the [CriticalFlaw Hud fixes](https://github.com/CriticalFlaw/TF2HUD.Fixes) and the [Eniere Improved Default HUD](https://wiki.teamfortress.com/wiki/User:Eniere/Improved_Default_HUD) but i believe this need to be properly suggested through an issue instead of a pull request instead.

### CODE REVIEW

**Comment by micwoj92** [comment]
Maybe wait until Rust will be in stable kernel release. I do not think it's worth trusting this language until then.

**Comment by getchoo** [comment]
+1 for kylul's viewmodel animations. i've been using them for a while and they're honestly a great alternative to paysus' (i think the only class animations missing is medic atm)

TF2 Bot Kicker also seems nice - especially since like you said, TF2BD doesn't support linux

File: docs/customization/see_also.md [hunk 1]
```diff
@@ -11,7 +11,10 @@ Here are a few awesome TF2 modifications:
   
 * [Paysus Animation Overhauls](https://steamcommunity.com/groups/PaysusSkins)
   — First person animation mods which greatly enhance the animations
-
+    
+    - [Kilul Animation packs](https://gamebanana.com/members/1655507)
+      — An another set of great first-person animation mods. Can be paired well with other Paysus Animation Overhauls for other classes.
```
Comment (getchoo) [inline docs/customization/see_also.md]: *Another set

File: docs/customization/see_also.md [hunk 2]
```diff
@@ -30,6 +33,9 @@ Here are a few awesome TF2 modifications:
   
 * [TF2 Bot Detector](https://github.com/PazerOP/tf2_bot_detector)
   — Automatically detects and votekicks cheaters/bots in TF2 casual
+    
+    - [TF2 Bot kicker gui](https://github.com/Bash-09/tf2-bot-kicker-gui)
+      — An alternative program, written in Rush, cross-platform. Suitable for Linux users.
```
Comment (getchoo) [inline docs/customization/see_also.md]: kind of weird wording, maybe something like:

> A cross-platform alternative, written in Rust (suitable for Linux users)

**Comment by mastercoms** [comment]
I won't be listing alternatives to Paysus' animations at this time.

As for the bot kicker, I feel like the value of bot kickers has been diminished overtime, and the one from Pazer is there mostly for legacy reasons. Furthermore, I feel like introducing an alternative bot kicker will cause confusion, especially considering the language in the description.

**Pull request closed without merging**